### PR TITLE
add SSR service/deployment

### PR DESCRIPTION
**Pull request by escattone**
This PR adds the React `ssr` Kubernetes `service` and `deployment` resources to MDN. This independently scalable service is used by the pods within the Kuma-based `web` `deployment` to handle React server-side-rendering requests.

This PR is meant as a companion to @davidflanagan's https://github.com/mozilla/kuma/pull/5362.

### CODE REVIEW

**Comment by escattone** [comment]
I've placed a `not ready` label on this for now until after I can test it tomorrow on stage along with mozilla/kuma#5362.

File: apps/mdn/mdn-aws/k8s/Makefile [hunk 1]
```diff
@@ -126,6 +140,7 @@ export KUMA_PROTOCOL ?= "https://"
 export KUMA_SECURE_HSTS_SECONDS ?= 0
 export KUMA_SERVE_LEGACY ?= True
 export KUMA_SESSION_COOKIE_SECURE ?= True
+export KUMA_SSR_TIMEOUT ?= 10
```
Comment (davidflanagan) [inline apps/mdn/mdn-aws/k8s/Makefile]: I think you had mentioned changing this to 5. Maybe you changed your mind. I don't think it matters much at all...

File: apps/mdn/mdn-aws/k8s/kumascript.deploy.yaml.j2 [hunk 2]
```diff
@@ -56,6 +56,8 @@ spec:
                 secretKeyRef:
                   name: newrelic-secrets
                   key: NEW_RELIC_LICENSE_KEY
+            - name: NODE_ENV
+              value: "production"
```
Comment (davidflanagan) [inline apps/mdn/mdn-aws/k8s/kumascript.deploy.yaml.j2]: `git grep NODE_ENV` in the kumascript repo doesn't turn up anything, so this seems safe, and a good catch to add this now for the future.

File: apps/mdn/mdn-aws/k8s/kumascript.deploy.yaml.j2 [hunk 3]
```diff
@@ -56,6 +56,8 @@ spec:
                 secretKeyRef:
                   name: newrelic-secrets
                   key: NEW_RELIC_LICENSE_KEY
```
Comment (davidflanagan) [inline apps/mdn/mdn-aws/k8s/kumascript.deploy.yaml.j2]: I forgot to mention that I wrote the ssr server to use new relic if NEW_RELIC_LICENSE_KEY and NEW_RELIC_APP_NAME are set, and I suspect that the logging information we get would be really valuable if we set them in the ssr.deploy.yaml.j2 file.  (not sure actually where the _APP_NAME gets set)

File: apps/mdn/mdn-aws/k8s/Makefile (hunk 1 above)
Comment (escattone) [reply apps/mdn/mdn-aws/k8s/Makefile]: Oh, good catch, yes!

**Comment by escattone** [comment]
Thanks for the review @davidflanagan! I agree, let's get the New Relic stuff set-up in another PR.

**Comment by escattone** [comment]
mozilla/kuma#5362 was deployed to stage, production, and standby earlier today. I manually deployed this PR to stage, tested it with the newly-deployed SSR code within mozilla/kuma#5362, and it worked great. I also manually deployed the `ssr` K8s service to stage, production, and standby (`make k8s-ssr-svc`), so that's ready for the next production push (K8s services are long-lived, and not deployed/updated as part of the rolling update done by a push).

I'm going to merge this as is, and @davidflanagan and I will inject the changes needed for New Relic monitoring of the `ssr` service in another PR.

**Pull request merged**